### DISCUSSION TITLE
Ability to disable broadcast prefix

### DISCUSSION THREAD

**Discussion by Artuto** [EssentialsX/Essentials#4464] [Ideas and Feature Suggestions] [answered]
Basically include a way to disable the /broadcast command prefix using something like a flag or a custom formatting code (this is what BungeeCord does with it's /alert command).

This would allow to send messages with custom prefixes, and when the flag is not provided it will append the prefix in the lang file.

**Comment by JRoy** [comment, marked as answer]
Use a custom locale to set the prefix to an empty string.
